database/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# Use PostgreSQL 17 base image
+FROM postgres:17 AS postgres_base
```

Comment:
Do you know how you would increase the number of connections from the default, as I think this might be needed.